lib/swagger-ui/swagger-ui.ts

```diff
@@ -0,0 +1,62 @@
+import * as swaggerUi from 'swagger-ui-dist';
+import { favIconHtml, htmlTemplateString, jsTemplateString } from './constants';
+import { OpenAPIObject, SwaggerCustomOptions } from '../interfaces';
+import { buildJSInitOptions } from './helpers';
+
+/**
+ * Used to create swagger ui initialization js file (
+ */
+export function buildSwaggerInitJS(
+  swaggerDoc: OpenAPIObject,
+  customOptions: SwaggerCustomOptions = {}
+) {
+  const { swaggerOptions, swaggerUrl } = customOptions;
+  const swaggerInitOptions = {
+    swaggerDoc,
+    swaggerUrl,
+    customOptions: swaggerOptions
+  };
+
+  const jsInitOptions = buildJSInitOptions(swaggerInitOptions);
+  return jsTemplateString.replace('<% swaggerOptions %>', jsInitOptions);
+}
+
+/**
+ * Stores absolute path to swagger-ui assets
+ */
+export const swaggerAssetsAbsoluteFSPath = swaggerUi.getAbsoluteFSPath();
+console.log(swaggerAssetsAbsoluteFSPath);
```

Comment:
```suggestion
console.log(swaggerAssetsAbsoluteFSPath);
```
Is this intentional? @flamewow

Reply:
nope, thanks for spotting it, forgot to remove it after debugging